Yona/Yona/Settings/SettingsViewController.swift

```diff
@@ -108,6 +96,12 @@ class SettingsViewController: UIViewController {
             UIApplication.sharedApplication().keyWindow?.rootViewController =  UINavigationController(rootViewController: welcome)
         }
     }
+    
+    //MARK: Navigation, Segue
+    override func shouldPerformSegueWithIdentifier(identifier: String, sender: AnyObject?) -> Bool {
+        //return false so we can load our detail data before pushing segue
+        return false
+    }
 }
 
```

Comment:
The same comment as the in PR #127 
Why do we create segues while we are not going to use them?